tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -450,9 +450,6 @@ WARNING:  continuous aggregate policy already exists for "mat_smallint"
 (1 row)
 
 \set ON_ERROR_STOP 1
-DROP MATERIALIZED VIEW mat_smallint;
-NOTICE:  drop cascades to 3 other objects
-DROP TABLE smallint_tab CASCADE;
 --bigint table
```

Comment:
We have tests for `SMALLINT (int2)` and `BIGINT (int8)`, shouldn't also have for `INTEGER (int4)` ??

Reply:
These tests are for code coverage only (policy_refresh_cagg_refresh_start_lt).  IIRC with the current code, one of the integer types is sufficient. Compression policies should be tested with both integer based and timestamp based materialized views. We have both kinds.